actors/builtin/power/power_state.go

```diff
@@ -86,7 +87,7 @@ func (st *State) minerNominalPowerMeetsConsensusMinimum(s adt.Store, miner addr.
 		return true, nil
 	}
 
-	// otherwise, if ConsensusMinerMinMiners miners meets min power requirement, return false
+	// otherwise, if ConsensusMinerMinMiners miners meet min power requirement, return false
 	if st.MinerAboveMinPowerCount > ConsensusMinerMinMiners {
```

Comment:
@ZenGround0 it seems to me like this should be changed to `>=`. What do you think?

Reply:
Yes, I'll submit a fix